final_task/rss_reader/Handler.py

```diff
@@ -0,0 +1,106 @@
+import feedparser
+from dataclasses import dataclass
+import html
+
+from Log import log_decore
+
+
+# convert from News class to json
+@log_decore
+def parse_to_json(news):
+    json_dict = {}
+    json_dict["title"] = news.title
+    json_dict["link"] = news.link
+    json_dict["date"] = news.date
+    json_dict["news"] = news.news
+    return json_dict
+
+
+# news kept
+@dataclass
+class News:
+    news: str
+    link: str
+    title: str
+    date: str
+    links: []
+
+
+class Handler:
+    @log_decore
+    # limit - count elements which user want see
+    def __init__(self, url, limit):
+
+        self.article = feedparser.parse(url)
+        self.numb_news = 0
+        self.parsers = []
+
+        # for every news, which user will see we create object
+        while self.numb_news < limit:
+            tmp_img_link = self.get_img_links(self.get_news(self.numb_news))
+            tmp_news = self.parse_html(self.get_news(self.numb_news))
+            t = News(tmp_news, url, self.get_title(self.numb_news), self.get_date(self.numb_news), tmp_img_link)
+            self.parsers.append(t)
+            self.numb_news += 1
```

Comment:
It is better for `__init__` to not contain complex logic. For example you can move it to a separate method